src/Dfc.CourseDirectory.WebV2/ServiceCollectionExtensions.cs

```diff
@@ -118,7 +118,7 @@ public static class ServiceCollectionExtensions
             services.AddFormFlow();
             services.AddJourneyStateTypes(typeof(ServiceCollectionExtensions).Assembly);
 
-            services.AddSingleton<IProviderOwnershipCache, ProviderOwnershipCache>();
+            services.AddTransient<IProviderOwnershipCache, ProviderOwnershipCache>();
```

Comment:
Why the change to transient (I think this is right but just checking)? Where does the cache actually live? I see it's injected.

Reply:
It now has a dependency on `ISqlQueryDispatcher` and that guy isn't `Singleton`